scripts/main/contextMenu.js

```diff
@@ -152,6 +152,34 @@ contextMenu.album = function (albumID, e) {
 	basicContext.show(items, e.originalEvent, contextMenu.close);
 };
 
+/**
+ *
+ * @returns {void}
+ * @param sourceAlbumID source album (which is being dragged)
+ * @param targetAlbumID target album (where it is dropped)
+ * @param {DragEvent} e
+ */
```

Comment:
```suggestion
/**
 * Handles drop event of an album onto an album and shows context menu to let the user pick the actions.
 * 
 * @param {string} sourceAlbumID source album (which is being dragged)
 * @param {string} targetAlbumID target album (where it is dropped)
 * @param {DragEvent} e
 *
 * @returns {void}
 */
```

Mainly the type annotation `{string}` was missing.